app/services/gias_csv_importer.rb

```diff
@@ -171,6 +186,17 @@ def determine_region(district_admin_code)
     end
   end
 
+  def associate_schools_to_trusts(trust_data)
+    Rails.logger.debug "Associating schools to trusts... "
+
+    trust_data.each do |uid, urns|
+      trust = Trust.find_by(uid:)
+      next unless trust
+
+      School.where(urn: urns).update_all(trust_id: trust.id)
```

Comment:
It's only syntactic sugar, but I think you can simplify this to:

```suggestion
      School.where(urn: urns).update_all(trust:)
```

Reply:
I don't think `update_all` accepts this.
As I replied in https://github.com/DFE-Digital/itt-mentor-services/pull/345#discussion_r1513041291, `update_all(region: inner_london)` returns a `TypeError: can't cast Region (TypeError)`

Reply:
Yeah, update_all needs the `id` here, but if I could use the nicer syntax I would!

Reply:
Oops. I missed @JamieCleare2525's reply on the other PR comment 🙃